### PR TITLE
add user to business

### PR DESCRIPTION
**Pull request by Automatic365**

### CODE REVIEW

**Comment by dtinianow** [comment]
Good refactoring of changing the role table to an enum for a user.  I think this will be more efficient for our purposes

File: app/models/user.rb [hunk 1]
```diff
@@ -1,10 +1,11 @@
...
   validates :username, presence: true, uniqueness: true
   validates :password, presence: true
+  validates :role, presence: true
 
   has_secure_password
 
   has_many :orders
 
-  enum role: %w(default admin)
+  enum role: %w(default business_admin platform_admin)
```
Comment (Yoyo2Code) [inline app/models/user.rb]: Good choice for enums.

**Comment by Yoyo2Code** [comment]
Way to test the validations after adding validations.

**Comment by dtinianow** [comment]
We set up a relationship for a business to belong to a user so we had to modify every factorygirl creation of a business to account for a user

**Pull request merged**